packages/grafana-data/src/valueFormats/dateTimeFormatters.ts

```diff
@@ -102,6 +102,9 @@ export function toSeconds(size: number, decimals?: DecimalCount, scaledDecimals?
   if (size === null) {
     return { text: '' };
   }
+  if (size === 0) {
```

Comment:
nitpick on formatting, needs a new line.
```suggestion
  
  if (size === 0) {
```

Reply:
added newline and comment.